### PR TITLE
New commands for tags

### PR DESCRIPTION
**Pull request by prakulkhebbur**
ill eventually add tags for all the commands in arduino reference

### CODE REVIEW

File: commands/tag.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ class TagCommand extends Command {
               tagFile.fields.forEach(field => {
                 tagEmbed.addField(field.name, field.value, false)
               })
+              .setFooter(field.footer)
```
Comment (max-bromberg) [inline commands/tag.js]: Right now we want to stick with a universal footer for all tags.

**Review by max-bromberg — CHANGES_REQUESTED** [review]
We're currently not interested in hand-translating all of the Arduino reference pages into bot tags. Instead, the helpers believe a simple quick hyperlink to the reference pages would suffice. Turning these reference tags into quick links is the way to go :)

File: commands/tag.js (hunk 1 above)
Comment (prakulkhebbur) [reply commands/tag.js]: ok 👍

**Comment by max-bromberg** [comment]
If this is still applicable, please open a new pull request off of the latest develop. Thanks for contributing.

**Pull request closed without merging**